### DISCUSSION TITLE
[Pathway Discussion]: WP5424 - How to refine / reorganize the pathway

### DISCUSSION THREAD

**Discussion by JuliaBatUM** [wikipathways/wikipathways-help#77] [pathway-discussion]
### Discussion

Hi!
I am not sure if this is the right location for adding what we discussed yesterday evening. Feel free to move the discussion to the right spot. 
Here is the screenshot of some difficulties I had with the map: 
![image](https://github.com/wikipathways/wikipathways-help/assets/130973169/489313e2-64ea-4c9d-861d-9c31338fbe31)
Point D) and E) should be fine.
I am also open to any other suggestion you have on the map.
Thanks! :)
Julia

### Type of revision

- [ ] Major
- [ ] Minor
- [ ] Deletion

**Comment by susancoort** [comment]
Dear Julia,
Once again many thanks for presenting your newly created pathway at the WikiPathways community meeting. You did a tremendous amount of work and it is amazing that all interactions are supported by literature. The discussion points are all very relevant and worthwhile to discuss with the WP community. Is it possible to create a separate discussion for your remaining questions? It would also be great if you could for questions B) and C) include a part of the pathway as image to help us to better understand the question you have.

**Reply by JuliaBatUM** [reply]
Dear Susan, 
Thanks ! :)
I will then create separate discussions under "Pathway Discussion" for the remaining questions mentioned above.
Kind regards,
Julia
